_pages/amarsurya.yaml

```diff
@@ -0,0 +1,3 @@
+ githubHandle: amarsurya
+ pageUrl: amarsurya.github.io
+ timestamp: 2018-01-10
```

Comment:
@amarsurya ensure you have `---` before and after the block of text and you should be set!